Rebasing-PRs.md

```diff
@@ -0,0 +1,85 @@
+We ask that PRs, in most cases, should be a single commit based on a reasonably recent commit
+(though not necessarily the latest) from the target branch, with no merge commits.
+
+
+# Getting started and creating a PR
+
+Following the usual Git convention, the following examples assume the remote referring to your fork [username/ChakraCore](https://github.com/username/ChakraCore), which you cloned from, is named `origin`, and [Microsoft/ChakraCore](https://github.com/Microsoft/ChakraCore) is named `upstream`.
```

Comment:
feels kinda weird to have a broken link to username/ChakraCore

Reply:
Good point. I'll unlink it and maybe write the URL as "code" like this:

`https://github.com/<username>/ChakraCore`